components/Sections/MainHeader.tsx

```
@@ -5,33 +5,108 @@ import { Button } from '@/components/ui/button';
+ * - Accessible markup with proper semantic structure
+ * - Customizable through HERO_CONFIG constant
+ */
+const MainHeader: React.FC<MainHeaderProps> = () => {
     return (
-        <section className="relative h-[60vh] md:h-[70vh] w-full flex items-center justify-center text-center text-white overflow-hidden">
-            <div className="absolute inset-0 z-0"> 
+        <header 
+          className={styles.hero}
+          aria-label="Main hero section"
+        >
+            {/* Background image with overlay */}
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)